### PR TITLE
Feat: product list page added

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
_Added products list
Please review PR

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -33,7 +33,7 @@
 .header {
     font-size: 2.5rem;
     font-weight: 700;
-    color: white;
+    color: blue;
```
Comment (ShoaibTamboli) [inline main.css]: please use CSS variables

File: main.css [hunk 2]
```diff
@@ -86,12 +86,15 @@
     font-size: var(--size-one);
     font-weight: bold;
 }
+a.login-btn {
+    color: white;
+}
```
Comment (ShoaibTamboli) [inline main.css:91]: Please use CSS variables

File: components/products/men-prod.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+   text-decoration: underline;
+}
+.filter-container {
+    display: flex;
+    flex-direction: column;
+}
+.filter-category {
+    
+    background-color: #1b1b1b;
+    color: white;
```
Comment (ShoaibTamboli) [inline components/products/men-prod.css]: Use CSS variable

File: components/products/men-prod.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+/* product card*/
+
+
+.product-container {
+    display: flex;
+    gap: 1rem;
+    flex-wrap: wrap;
+    width: 100%;
+}
```
Comment (ShoaibTamboli) [inline components/products/men-prod.css]: include justify content : center, it will add some space

File: components/products/men-prod.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    max-width: 20rem;
+    height: 22rem;
+    max-height: 24rem;
+  }
+  .card-def {
+      color: var(--red);
+  }
+  
+  .card-header>h1 {
+    color: darkviolet;
```
Comment (ShoaibTamboli) [inline components/products/men-prod.css:81]: use CSS variable

**Review by ShoaibTamboli — APPROVED** [review]
Please use CSS Variables for color.

**Review by Asuraking1n — COMMENTED** [review]
Looks fine to me....just use colour variables

File: main.css (hunk 1 above)
Comment (anurag-kkumarr) [reply main.css]: done

File: components/products/men-prod.css (hunk 4 above)
Comment (anurag-kkumarr) [reply components/products/men-prod.css]: Thanks, It worked.

**Comment by anurag-kkumarr** [comment]
Thanks for PR review.

**Pull request merged**